docs/ref-web-components-overview.md

```diff
@@ -1,33 +1,35 @@
----
-id: ref-web-components-overview
-title: Overview
----
-
-Accord Project publishes [React](https://reactjs.org) user interface components for use in web applications. Please refer to the web-components project [on GitHub](https://github.com/accordproject/web-components) for detailed usage instructions.
-
-You can preview these components in [the project's storybook](https://ap-web-components.netlify.app/).
-
-## Contract Editor
-
-The Contract Editor component provides a rich-text content editor for contract text with embedded clauses.
-
-> Note that the contract editor does not currently support the full expressiveness of Cicero Templates. Please refer to the Limitations section for details.
-
-### Limitations
-
-The contract editor does not support templates which use the following CiceroMark features:
-
-* Lists containing [nested lists](markup-commonmark#nested-lists)
-* Templates [list blocks](markup-blocks#list-blocks)
-
-## Error Logger
-
-The Error Logger component is used to display structured error messages from the Contract Editor.
-
-## Navigation
-
-The Navigation component displays an outline view for a contract, allowing the user to quickly navigate between sections.
-
-## Library
-
-The Library component displays a vertical list of library item metadata, and allows the user to add an instance of a library item to a contract.
+---
+id: ref-web-components-overview
+title: Overview
+---
+
+Accord Project publishes [React](https://reactjs.org) user interface components for use in web applications. Please refer to the web-components project [on GitHub](https://github.com/accordproject/web-components) for detailed usage instructions.
+
+You can preview these components in [the project's storybook](https://ap-web-components.netlify.app/).
+
+![Template-Studio-V2](/docs/assets/reference/tsv2.png)

```

Comment:
Although I'm not a maintainer I apologize for commenting here but, do we need this image which is of `template-studio-v2` (which is archived now)?
Link of this PR preview build : https://deploy-preview-354--accordproject-old.netlify.app/docs/next/ref-web-components-overview.html

Reply:
Yeah..I need to update the image!!

Reply:
I think we don't really need an image here at the moment, so this could be removed.